db/migrate/20160104131622_remove_deprecated_columns.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveDeprecatedColumns < ActiveRecord::Migration
+  def change
+    remove_column :subject_workflow_counts, :set_member_subject_id
+    remove_column :classifications, :subject_ids
```

Comment:
The association is already a `has-many through` so yes, reading and even assignment works fine.